deviceModeInit.js

```diff
@@ -73,7 +84,14 @@ var rudderTracking = (function () {
       userId &&
       cookie_action({ action: "get", name: "rudder_user_id" }) !== "captured"
     ) {
-      rudderanalytics.identify(String(userId));
+      if (heapCookieObject) {
+        rudderanalytics.identify(String(userId), {
+          heapUserId: heapCookieObject.userId,
+          heapSessionId: heapCookieObject.sessionId
+        });     
+      } else {
+        rudderanalytics.identify(String(userId));
+      }
```

Comment:
Many times the user will be anonymous user then the identify call won't be triggered. Can we add the heap userId and SessionId in track call as well

Reply:
Addressed.
A single identify call for a not logged anonymous user is made.